CHANGELOG.md

```diff
@@ -7,122 +7,74 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-- Fix Snaps view search ([#14693](https://github.com/MetaMask/metamask-extension/pull/14693))
-- Remove gas editing in Swaps ([#14673](https://github.com/MetaMask/metamask-extension/pull/14673))
-- Remove confirmation for terminated snaps ([#14566](https://github.com/MetaMask/metamask-extension/pull/14566))
-- Chromedriver v101 ([#14617](https://github.com/MetaMask/metamask-extension/pull/14617))
-- snaps-skunkworks@0.12.0 ([#14670](https://github.com/MetaMask/metamask-extension/pull/14670))
-- Fix auto-lock e2e test ([#14685](https://github.com/MetaMask/metamask-extension/pull/14685))
-- Fix eth_sign 'Learn more' dialog link ([#14667](https://github.com/MetaMask/metamask-extension/pull/14667))
-- Updating alert text color ([#14674](https://github.com/MetaMask/metamask-extension/pull/14674))
-- Auto lock e2e ([#14624](https://github.com/MetaMask/metamask-extension/pull/14624))
-- lavamoat - bump for stats support ([#14641](https://github.com/MetaMask/metamask-extension/pull/14641))
-- Fix Ropsten Test Network icon ([#14626](https://github.com/MetaMask/metamask-extension/pull/14626))
-- TransactionListItem: fix save/cancel rounded btn ([#14659](https://github.com/MetaMask/metamask-extension/pull/14659))
-- Add New Tokens Found page ([#14366](https://github.com/MetaMask/metamask-extension/pull/14366))
-- Swaps / STX improvements ([#14622](https://github.com/MetaMask/metamask-extension/pull/14622))
-- Simplify removeSnap functionality after snaps-skunkworks update ([#14646](https://github.com/MetaMask/metamask-extension/pull/14646))
-- Fixing Speed up/Cancel information text ([#14551](https://github.com/MetaMask/metamask-extension/pull/14551))
-- Fix SRP paste duplication ([#14625](https://github.com/MetaMask/metamask-extension/pull/14625))
-- LavaMoat - UI upgrade - secure package naming ([#14565](https://github.com/MetaMask/metamask-extension/pull/14565))
-- E2e send tests ([#14568](https://github.com/MetaMask/metamask-extension/pull/14568))
-- slip44@2.1.0 ([#14619](https://github.com/MetaMask/metamask-extension/pull/14619))
-- Reproducible `.zip` files ([#14623](https://github.com/MetaMask/metamask-extension/pull/14623))
-- fix incorrect casing of includeUsdRate option passed to CurrencyRateController ([#14615](https://github.com/MetaMask/metamask-extension/pull/14615))
-- Fix redundant entry in lockfile ([#14616](https://github.com/MetaMask/metamask-extension/pull/14616))
-- Jest tests - incremental coverage ([#14612](https://github.com/MetaMask/metamask-extension/pull/14612))
-- build - update bify-module-groups for build determinism ([#14610](https://github.com/MetaMask/metamask-extension/pull/14610))
+- Add Coinbase Pay as fiat onramp option ([#14648](https://github.com/MetaMask/metamask-extension/pull/14648))
+- Add networks without ticker trait, for better analytics ([#14607](https://github.com/MetaMask/metamask-extension/pull/14607))
```

Comment:
> Add networks without ticker trait, for better analytics ([#14607](https://github.com/MetaMask/metamask-extension/pull/14607))

Can be removed, because it is only a changed to metrics, which is not noticeable to the end user

Reply:
same for
> Add event tracking for Token Detection V2 ([#14441](https://github.com/MetaMask/metamask-extension/pull/14441))

Reply:
noted. Ty